### ISSUE TITLE
Make regex more robust for checking URL

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#997]
We need to improve regex for URL in "plugin_header_invalid_plugin_uri_domain". Following was not detected:

- https://example.com
- https://www.example.com/

Ref:

```php
preg_match( '/\/\/(example\.com|example\.net|example\.org)\//', $plugin_header['PluginURI'], $matches )
```

Also, this check seems to be missing for "Author URI" header.

**Comment by luckaty** [comment]
Hi! I'd love to work on this issue. Please assign me if it's still available 🙌

**Comment by ernilambar** [comment]
PR https://github.com/WordPress/plugin-check/pull/998 has been already prepared for this issue.
